app/src/main/java/com/kamron/pogoiv/AutoAppraisal.java

```diff
@@ -148,7 +148,10 @@ public void reset() {
      * @param hash the hash of the bitmap used by ocr
      */
     private void addInfoFromAppraiseText(String appraiseText, String hash) {
-        boolean match = setIVRangeWith(appraiseText);
```

Comment:
Yea I think its best to either revert, or make clear comments so we know whats going on.